hikari/api/special_endpoints.py

```diff
@@ -417,3 +422,142 @@ def add_voice_channel(
+        """
+
+    @mentions_everyone.setter
+    def mentions_everyone(self, mentions: undefined.UndefinedOr[bool] = undefined.UNDEFINED, /) -> None:
+        raise NotImplementedError
+
+    @property
+    @abc.abstractmethod
+    def user_mentions(
+        self,
+    ) -> undefined.UndefinedOr[typing.Union[snowflakes.SnowflakeishSequence[users.PartialUser], bool]]:
+        """Whether and what user mentions should be enabled for this response.
+
+        Returns
+        -------
+        hikari.undefined.UndefinedOr[typing.Union[hikari.snowflakes.SnowflakeishSequence[hikari.users.PartialUser], builtins.bool]]
+            Either a sequence of object/IDs of the users mentions should be enabled for,
+            `False` or `hikari.undefined.UNDEFINED` to disallow any user mentions
+            or `True` to allow all user mentions.
+        """  # noqa: E501 - Line too long
+
+    @user_mentions.setter
+    def user_mentions(
+        self,
+        mentions: undefined.UndefinedOr[
+            typing.Union[snowflakes.SnowflakeishSequence[users.PartialUser], bool]
+        ] = undefined.UNDEFINED,
+        /,
+    ) -> None:
+        raise NotImplementedError
+
+    @property
+    @abc.abstractmethod
+    def role_mentions(
+        self,
+    ) -> undefined.UndefinedOr[typing.Union[snowflakes.SnowflakeishSequence[guilds.PartialRole], bool]]:
+        """Whether and what role mentions should be enabled for this response.
+
+        Returns
+        -------
+        hikari.undefined.UndefinedOr[typing.Union[hikari.snowflakes.SnowflakeishSequence[hikari.users.PartialUser], builtins.bool]]
+            Either a sequence of object/IDs of the roles mentions should be enabled for,
+            `False` or `hikari.undefined.UNDEFINED` to disallow any role mentions
+            or `True` to allow all role mentions.
+        """  # noqa: E501 - Line too long
+
+    @role_mentions.setter
+    def role_mentions(
+        self,
+        mentions: undefined.UndefinedOr[
+            typing.Union[snowflakes.SnowflakeishSequence[guilds.PartialRole], bool]
+        ] = undefined.UNDEFINED,
+    ) -> None:
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def add_embed(self, embed: embeds.Embed, /) -> None:
+        """Add an embed to this response.
+
+        Parameters
+        ----------
+        embed : hikari.embeds.Embed
+            Object of the embed to add to this response.
+        """
+
+    @abc.abstractmethod
+    def build(self, entity_factory: entity_factory_.EntityFactory, /) -> data_binding.JSONObject:
+        """Build a JSON object from this builder.
+
+        Parameters
+        ----------
+        entity_factory : hikari.api.entity_factory.EntityFactory
+            The entity factory to use to serialize entities within this builder.
+
+        Returns
+        -------
+        hikari.internal.data_binding.JSONObject
+            The built json object representation of this builder.
+        """
```

Comment:
This doesnt need to be a special endpoint, it can just be a normal one which takes in `embed` and `embeds`

Reply:
except this isn't an endpoint you call, this is the result that's used as a response within a rest server (you are the endpoint)

Reply:
Well, for the methods that are here, this doesnt need to be a builder, it can be a normal call

Reply:
A call to/on what?